modules/core/output_modules.php

```diff
@@ -218,13 +256,27 @@ protected function output() {
             ' <label for="stay_logged_in">'.$this->trans('Stay logged in').'</label></div>';
         }
         if (!$this->get('router_login_state')) {
-            return '<h1 class="title">'.$this->html_safe($this->get('router_app_name', '')).'</h1>'.
+            $config = new Hm_Site_Config_File(CONFIG_FILE);
+            $fancy_login = $config->get('fancy_login','n');
+            if($fancy_login=='n'):
```

Comment:
Can you make sure to keep the old login as default and only use the new one in case fancy_login = 'y'?

Reply:
Yes, for now, defaultly the setting load with 'n' (even when omitted from hm3.ini)